src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -2560,6 +2562,8 @@ internal class TestingStuff
             internal Action<Index> BeforeIndexThreadExit;
             internal Action<Index> BeforeIndexStart;
 
+            public Action BeforeHandleDatabaseRecordChange;
```

Comment:
Why is it public? Everything is either internal or private here

Reply:
done